configure.ac

```diff
@@ -29,7 +29,11 @@ AS_IF([test "x$enable_bzip2" != "xno" ],
 	 AC_CHECK_LIB([bz2], [BZ2_bzBuffToBuffCompress], [], [AC_MSG_ERROR([the libbz2 library is missing])])],
   [AC_DEFINE(SWUPD_WITHOUT_BZIP2,1,[Do not use bzip2 compression])]
 )
-
+AC_ARG_ENABLE(
+  [signatures],
```

Comment:
The documentation on the line below is correct now, but this argument needs updating too.